README.md

```diff
@@ -64,7 +70,14 @@ import { getConfig } from '@imtbl/core-sdk';
 const ethNetwork = 'ropsten'; // or mainnet;
 
 // Use the helper function to get the config
-const config = getConfig(ethNetwork);
+const config = getConfig({
```

Comment:
Is `Config` the right name for this type, or are we actually defining `Environments`, or something similar.

Reply:
Not Environments as it's always supposed to return a single environment's configs.